docs/_blog/2020-05-14-codewind-in-intellij-may.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "Codewind in IntelliJ IDEA"
+categories: blog
+author_picture: images/blog/author_icon_john_pitman.png
+author_url:
+author: John Pitman
+blog_description: "Update on support for Codewind in IntelliJ IDEA"
+permalink: codewind-in-intellij-may
+duration: 2 minutes
+tags: [IntelliJ]
+---
+
+Over the past few months, we've been hard at work creating a Codewind plugin for IntelliJ IDEA. This is a tech preview, because there are still some fundamental features to be implemented, but we have made significant progress.
+
+Our [first tech-preview release](https://www.eclipse.org/codewind/introducing-eclipse-codewind-on-intellij.html){:target="_blank"} included basic support for Codewind, and last month we added the ability to show the log files, open the performance dashboard, and debug projects.
+
+This month, we've added the support for opening the application metrics dashboard, and the ability to open a shell into the application container.  Stay tuned for more function to be delivered next month.
```

Comment:
- Remove comma after "dashboard"
- I'd prefer to remove the sentence, "Stay tuned...". I'm kinda wary about anything that sounds like a promise for the future.

Reply:
Fixed